### PR TITLE
Navigation Page: Fast Link Buttons and Improved UI for setting dates

### LINKED ISSUE #557: Add button on navigation page that fast links certain actions
ex: "release now" to change grading due date to now.

### LINKED ISSUE #566: Improve UI for setting dates
To be done for this issue:
- [x] Have JS check date consistency (when hitting submit button) when creating/editing a gradeable
- [x] Quick action buttons on Navigation Page ("Release Grades Now", etc.)
- [x] Have a "now" and "infinity" (set to 12/31/9999) buttons next to each date field
- [x] Four digit year format on all date fields (navigation page, show submission, gradeable form, etc.)

-----
Allow an never/infinity option, especially for grades.

Add some quick set features from the main page like "release grades now", "open homework now".

Also, inside of the full gradeable form, next to each date, have a "now" button?  
(rough idea: design needs to be thought through)
And/or default to "now"?

### PR DESCRIPTION
**Pull request by AndrewMRPI**
There are now fast link buttons on the navigation page. There is also an infinity button for the date picker. Also the years now display 4 digits. 

Closes #566
Closes #557

### CODE REVIEW

File: site/app/libraries/database/DatabaseQueriesPostgresql.php [hunk 1]
```diff
@@ -1090,28 +933,131 @@ public function updateGradeable($details) {
...
         $this->database->query("UPDATE sessions SET session_expires=(current_timestamp + interval '336 hours') 
         WHERE session_id=?", array($session_id));
     }
 
+    public function getAllGradeablesIdsAndTitles() {
+        $this->database->query("SELECT g_id, g_title FROM gradeable ORDER BY g_title ASC");
+        return $this->database->rows();
+    }
+    
+
```
Comment (wangj35) [inline site/app/libraries/database/DatabaseQueriesPostgresql.php]: This same function exists on line 1084, which was causing some problems.

File: site/app/controllers/AdminController.php [hunk 2]
```diff
@@ -8,6 +8,7 @@
 use app\controllers\admin\AdminGradeableController;
 use app\controllers\admin\ConfigurationController;
 use app\controllers\admin\UsersController;
+use app\controllers\admin\AdminGradeableController;
```
Comment (wangj35) [inline site/app/controllers/AdminController.php]: lines 8 and 11 are duplicates

File: site/app/views/NavigationView.php [hunk 3]
```diff
@@ -14,6 +14,7 @@ public function showGradeables($sections_to_list) {
         $semester = $this->core->getConfig()->getSemester();
         $course = $this->core->getConfig()->getCourse();
         $site_url = $this->core->getConfig()->getSiteUrl();
+        $columnSpan = 10;
```
Comment (wangj35) [inline site/app/views/NavigationView.php]: what is the point of this variable? I don't think it's referenced in the page anywhere.

Comment (AndrewMRPI) [inline site/app/views/NavigationView.php]: Well I tried doing that and it didn't work so I just got frustrated and hard coded 10 in. I forgot to rid of the variable though.

Comment (wangj35) [inline site/app/views/NavigationView.php]: Ah, I see.

**Comment by wangj35** [comment]
Based off the meeting conversation I'll re-implement the total points row for Numeric gradeables on this branch.

**Pull request merged**